samples/sp_whoIsActive/tasks/config.js

```diff
@@ -0,0 +1,27 @@
+var path = require('path');
+
+var projectRoot = path.resolve(path.dirname(__dirname));
+var srcRoot = path.resolve(projectRoot, 'src');
+var viewsRoot = path.resolve(srcRoot, 'views');
+var htmlcontentRoot = path.resolve(viewsRoot, 'htmlcontent');
+var outRoot = path.resolve(projectRoot, 'out');
+var htmloutroot = path.resolve(outRoot, 'src/views/htmlcontent');
```

Comment:
You don't have htmlContent do you? Can you clean up the stuff from the mssql extension that's no longer relevant?